website/docs/docs/dbt-versions/product-lifecycles.md

```diff
@@ -14,7 +14,7 @@ Any dbt feature will fall into one of the following lifecycle states:
 
 ### dbt Cloud
 
-- **Beta:** Beta features may be made available for the purpose of customer testing and evaluation. These may not be feature-complete or fully stable. There may still be some planned additions and modifications to product behaviors while in Beta. Breaking changes may occur – although we will do our best to communicate them in advance, we may not always be able to do so. Beta features may not be fully documented, technical support may be limited, and service level objectives (SLOs) may not be provided.
+- **Beta:** Beta features may be made available for the purpose of customer testing and evaluation. These may not be feature-complete or fully stable. There may still be some planned additions and modifications to product behaviors while in Beta. Breaking changes may occur – although we will do our best to communicate them in advance, we may not always be able to do so. Beta features may not be fully documented, technical support may be limited, and service level objectives (SLOs) may not be provided. Download the [Beta Features Terms and Conditions](/assets/beta-tc.pdf) for more info.
```

Comment:
sugg for "may" vs "might" : 

i've been trained to use "may" when referring to permission (like, you _may_ sit down) and "might" when referring to "maybe" for translation purposes. "may" has many uses and that ambiguity can be harder to translate/harder to understand.

```suggestion
- **Beta:** Beta features may be made available for the purpose of customer testing and evaluation. These might not be feature-complete or fully stable. There might still be some planned additions and modifications to product behaviors while in beta. Breaking changes could occur &mdash; although we will do our best to communicate them in advance, we might not always be able to do so. Beta features might not be fully documented, technical support might be limited, and service level objectives (SLOs) might not be provided. Download the [Beta Features Terms and Conditions](/assets/beta-tc.pdf) for more details.
```